packages/bson-transpilers/package.json

```diff
@@ -12,6 +12,7 @@
   "main": "index.js",
   "compass:main": "index.js",
   "scripts": {
+    "bootstrap": "npm run compile",
```

Comment:
I’m wondering if there’s a reason this isn’t part of bootstrapping (maybe because it involves a fairly complex java build step)?

Reply:
CI just failed because it was unable to locate a Java Runtime, so I removed bootstrapping.